applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h

```diff
@@ -558,23 +591,13 @@ namespace Kratos {
         //*************************************************************************************
         //*************************************************************************************
 
-        void InitializeNonLinIteration(ModelPart& r_model_part,
-                                               TSystemMatrixType& A,
-                                               TSystemVectorType& Dx,
-                                               TSystemVectorType& b) override
+        void FinalizeNonLinIteration(ModelPart &rModelPart, TSystemMatrixType &A, TSystemVectorType &Dx, TSystemVectorType &b) override
         {
-            KRATOS_TRY
+            ProcessInfo& CurrentProcessInfo = rModelPart.GetProcessInfo();
```

Comment:
```suggestion
            const ProcessInfo& r_current_process_info = rModelPart.GetProcessInfo();
```
I know it was already like this, but at least the const we should add.